### PR TITLE
Add beheading and removal of C*lumbus statue in Boston MA

### PR DESCRIPTION
**Pull request by noahsbwilliams**

### CODE REVIEW

**Review by KungRaseri — APPROVED** [review]

File: content/post/columbus-bos.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Beheaded Christopher Columbus Statue In Boston Will Be Removed From North End Park"
+date: 2020-06-10T16:09:33-08:00
+draft: true
+---
+
+The statue of genocidal pillager Christopher Columbus [is to be removed today](https://boston.cbslocal.com/2020/06/10/christopher-columbus-statue-beheaded-boston-massachusetts/amp/?__twitter_impression=true) after being [beheaded overnight](https://www.nbcnews.com/news/us-news/boston-christopher-columbus-statue-beheaded-richmond-statue-thrown-lake-n1229201).
```
Comment (psivesely) [inline content/post/columbus-bos.md:14]: You can remove everything after `https://boston.cbslocal.com/2020/06/10/christopher-columbus-statue-beheaded-boston-massachusetts/`.

Comment (noahsbwilliams) [reply content/post/columbus-bos.md:14]:
```suggestion
The statue of genocidal pillager Christopher Columbus [is to be removed today](https://boston.cbslocal.com/2020/06/10/christopher-columbus-statue-beheaded-boston-massachusetts/).
```

Ok, sure 👍

**Comment by Gorcenski** [comment]
I've added a second post now that it's down but I'm going to merge this in manually!

**Comment by Gorcenski** [comment]
Thank you!!

**Pull request closed without merging**